pom.xml

```diff
@@ -12,12 +12,11 @@
 
 	<groupId>org.isf</groupId>
 	<artifactId>OH-core</artifactId>
-	<packaging>jar</packaging>
-	<version>1.13.0-SNAPSHOT</version>
+	<version>1.14.0-SNAPSHOT</version>
 
 	<properties>
-		<oh.version>1.13.0-SNAPSHOT</oh.version>
-		<java.version>11</java.version>
+		<oh.version>${project.version}</oh.version>
```

Comment:
This substitution doesn't take place.  If you do `mvn clean install` and look at the `OH-core-1.14.0-SNAPSHOT.pom` in the local repo you will see:
```java
	<properties>
		<oh.version>${project.version}</oh.version>
		<java.version>17</java.version>

``` 
We need to hard code the value instead of trying to use substitution.